content/cloud-docs/api-docs/configuration-versions.mdx

```diff
@@ -346,6 +346,36 @@ curl \
   https://archivist.terraform.io/v1/object/4c44d964-eba7-4dd5-ad29-1ece7b99e8da
 ```
 
+## Download Configuration Files
+
+`GET /configuration-versions/:id/download`
```

Comment:
This is using wording adjusted from the [download plan export API](https://www.terraform.io/cloud-docs/api-docs/plan-exports#download-exported-plan-data). 

This is also formatted similarly to the [retrieve json execution plan](https://www.terraform.io/cloud-docs/api-docs/plans#retrieve-the-json-execution-plan) endpoints as there are two endpoints that alias to the same functionality there as well as here.